package-build/package-build.el

```diff
@@ -146,6 +147,10 @@ function for access to this function")
     (:exclude ".dir-locals.el" "test.el" "tests.el" "*-test.el" "*-tests.el"))
   "Default value for :files attribute in recipes.")
 
+(defun mappend (fn seq)
+  "Apply `FN' to the `SEQ', and append results."
+  (apply #'append (mapcar fn seq)))
+
```

Comment:
`cl-mapcon` would do this too, I think. You definitely shouldn't introduce new unprefixed helper functions.